bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/MieleGatewayCommunicationController.java

```diff
@@ -83,6 +85,7 @@ public JsonElement invokeRPC(String methodName, Object[] args) throws MieleRpcEx
 
         String requestBody = requestBodyAsJson.toString();
         Request request = httpClient.newRequest(uri).method(HttpMethod.POST)
+                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MICROSECONDS)
```

Comment:
```suggestion
                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS)
```

(or better yet, use seconds)

Reply:
Fixed.